### PR TITLE
Feature/adding profile

### PR DESCRIPTION
**Pull request by gwkim8107**
Adding Jayden pics and bio.

### CODE REVIEW

File: src/js/ApiService.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ class ApiService {
 			legislationToCases {
```
Comment (dlouise64) [inline src/js/ApiService.js]: @gwkim8107 Can move these APIService changes into a new pull request as this one is just for adding a profile :)

File: src/js/ApiService.js [hunk 2]
```diff
@@ -24,7 +24,8 @@ class ApiService {
...
 				section,
 				legislation {
-					title
+					title,
```
Comment (dlouise64) [inline src/js/ApiService.js]: @gwkim8107 Can move these APIService changes into a new pull request as this one is just for adding a profile :)

File: src/jsx/components/SingleCaseView.jsx [hunk 3]
```diff
@@ -170,8 +170,8 @@ const SingleCaseView = props => {
 								<tbody className="">
 									{legislationToCases.map((obj, i) => (
 										<tr className="" key={`legislation-reference-${i}`}>
-											<td className="">{obj.legislation.title}</td>
-											<td className="">{obj.section}</td>
+											<a href={`http://legislation.govt.nz${obj.legislation.link}`} target='_blank'><td className="">{obj.legislation.title}</td>
```
Comment (dlouise64) [inline src/jsx/components/SingleCaseView.jsx]: @gwkim8107 same here, move into a new pull request. Also, something tiny to change for consistency. `target='_blank'` should use double quotes 👍

File: src/jsx/pages/About.jsx [hunk 4]
```diff
@@ -229,7 +230,18 @@ class About extends Component {
...
+									/>
+								</div>
+								<div className="card-item-small">
+									<div>
+										<img src={Jayden} alt="Gwangwoo Jayden Kim" />
+										<strong>Jayden Kim</strong>
+									</div>
+									<ReadMore
+										text="
+										Jayden has worked as a backend developer,but now he is trying to extend his ability to frontend area, especially using React JS. Apart from that he would like to support OpenLaw NZ project, and he even hopes his frontend skills to improve further.
```
Comment (dlouise64) [inline src/jsx/pages/About.jsx]: @gwkim8107 Nice profile Jayden :) Good to see you are wanting to learn React!

I've added some small grammatical fixes for you here:

`Jayden has worked as a backend developer, but now he is trying to extend his ability to frontend area, especially using ReactJS. Apart from that, he would like to support the OpenLaw NZ project.`

Don't really need `, and he even hopes his frontend skills to improve.` as you've mentioned you want to extend your ability in the first sentence already :)

File: src/jsx/pages/About.jsx [hunk 5]
```diff
@@ -25,6 +25,7 @@ import Infra from "../../img/about-us/openlaw-infrastructure.png";
 import Nicolas from "../../img/about-us/Nicolas-zhou.png";
 import Jins from "../../img/about-us/Jins_Mary_Job.png";
 import Dana from "../../img/about-us/dana-iti.png";
+import Jayden from "../../img/about-us/jayden-kim.png";
```
Comment (dlouise64) [inline src/jsx/pages/About.jsx]: @gwkim8107 just need to add the image to the project. Lina can do this for you.

File: src/js/ApiService.js (hunk 2 above)
Comment (gwkim8107) [reply src/js/ApiService.js]: Yes, I agree with that. I am intending to add this code here.

**Pull request closed without merging**